sources/packages/web/src/services/http/InstitutionLocation.ts

```diff
@@ -92,4 +57,17 @@ export class InstitutionLocationApi extends HttpBaseClient {
       ),
     );
   }
+
+  /**
+   * Get location details of logged in user.
+   * @returns location details.
+   */
+  async getMyInstitutionLocationsDetails(
+    header?: any,
```

Comment:
since you commented this function. can we add `header in the comment

Reply:
I am removing the header from these methods. This is a leftover. I was not able to find a reason to keep them.

Reply:
Changed!